src/System.Security.Cryptography.Pkcs/tests/Certificates.cs

```diff
@@ -0,0 +1,389 @@
+                + "74c2a20dd9e07e149e7ab52157db748160ad81f91019297baa58ce68656b0b2f7c9ac88b3da6920c2a5eab7bcc2629974f8a"
+                + "6c8bf33629af05e4e34d5d24393448e9751b7708f5915b0fd97a5af4dd5a37d71b18b6526316cbc65b1c6af8a6779acbc470"
+                + "2381f027bdb118cb84e9005b02a8bd2d02365d280cffb04831f877de7bd3d3287f11beed8978a5389e2b28317eb90569781f"
+                + "94f66f672736a09b4a7caeaaefd1909f2d20255df51512dbd08ec6125455d932b626bdfd3c4f669148fa783671f90b59ceff"
+                + "560c338f92cbe8bf7fbab4db3e9b943effac747eb34f06bd72aee961ed31742caa2a9934a5fe4685677ecbca6fb1b1c0b642"
+                + "b4f71d55d0e2cb1dc10ce845514090cc117a875c4d10c0ce367e31091144eacd7e600792d61d036bde020e3bb9a004a7dd1a"
+                + "cf03541b6fff3bcef4c30df05d98b75688320685261b2b34813407b20a7c92a04eeb46cb7e618a6ee32154728ba6735668f4"
+                + "11abece4ba07426a394b3181af301306092a864886f70d0109153106040401000000303906092a864886f70d010914312c1e"
+                + "2a005200530041005300680061003500310032004b00650079005400720061006e00730066006500720031305d06092b0601"
+                + "04018237110131501e4e004d006900630072006f0073006f0066007400200053006f0066007400770061007200650020004b"
+                + "00650079002000530074006f0072006100670065002000500072006f007600690064006500723082023306092a864886f70d"
+                + "010701a0820224048202203082021c30820218060b2a864886f70d010c0a0103a08201f0308201ec060a2a864886f70d0109"
+                + "1601a08201dc048201d8308201d43082013da00302010202102f5d9d58a5f41b844650aa233e68f105300d06092a864886f7"
+                + "0d01010d05003020311e301c060355040313155253415368613531324b65795472616e7366657231301e170d313630343138"
+                + "3131303532355a170d3137303431383137303532355a3020311e301c060355040313155253415368613531324b6579547261"
+                + "6e736665723130819f300d06092a864886f70d010101050003818d0030818902818100b2eca20240da8486b1a933ade62ad8"
+                + "781ef30d4434ebbc9b3fc9c550d0f9a75f4345b5520f3d0bafa63b8037785d1e8cbd3efe9a22513dc8b82bcd1d44bf26bd2c"
+                + "292205ca3e793ff1cb09e0df4afefb542362bc148ea2b76053d06754b4a37a535afe63b048282f8fb6bd8cf5dc5b47b75027"
+                + "60587f84d9995acbf1f34a3ca10203010001a30f300d300b0603551d0f040403020520300d06092a864886f70d01010d0500"
+                + "03818100493d857684d27468dd09926d20933254c7c79645f7b466e7b4a90a583cedba1c3b3dbf4ccf1c2506eb392dcf15f5"
+                + "3f964f3c3b519132a38eb966d3ea397fe25457b8a703fb43ddab1c52272d6a12476df1df1826c90fb679cebc4c04efc764fd"
+                + "8ce3277305c3bcdf163791784d778663194097180584e5e8ab69039908bf6f863115301306092a864886f70d010915310604"
+                + "0401000000303b301f300706052b0e03021a04142a5f75a3a633982a7110846b8ea3298abef778530414669cb82c9cc3ceb4"
+                + "d3ca9f65bd57ba82961660d9020207d0").HexToByteArray();
+
+            public static byte[] s_DfHelleKeyAgreement1Cer =
```

Comment:
I’ll rename to DHKeyAgree1 since that’s the actual Issued-To on the cert, and all the others are named to match the Issued-To.